index.html

```diff
@@ -2375,16 +2375,33 @@ <h2>
         |params|.
         </li>
       </ol>
+    </section>
+    <section>
+      <h2>
+        Launch queue and launch params
```

Comment:
I think the "Launch queue and launch params" section should come before the file handler sections since file handling is a "client" of these definitions.

Reply:
Done